english/more_cowbell/more_cowbell.md

```diff
@@ -4,3 +4,6 @@ He has a [pocket watch](https://www.youtube.com/watch?v=YFtHjV4c4uw) in his hand
 
 He also hands you a [weapon](https://www.youtube.com/watch?v=wCDIYvFmgW8)
 
+He doesn't care about time or weapons though. 
+He wants to become a new type of man, one that trancends the current reality and feels time as though it is water and weapons as though they are air. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.